.github/workflows/cmake.yml

```diff
@@ -50,6 +50,17 @@ jobs:
         working-directory: ${{github.workspace}}/dotnet
         run: dotnet build /p:EnableWindowsTargeting=true
 
-      - name: Run .NET tests
-        working-directory: ${{github.workspace}}/dotnet
-        run: dotnet test /p:EnableWindowsTargeting=true
```

Comment:
Removing because it messes with the coverage results of the C++ tests below?

Reply:
Good catch, no the removal was not intended and there is no problem with running the tests here as far as I can tell. Thanks a lot!